CONTRIBUTING.md

```diff
@@ -335,7 +336,28 @@ since the latest release over the hotfixed `master`.
 This would mean, however, that the commits referenced from PRs previously merged to `develop` will no longer be part of `develop`'s history,
 which is rather unacceptable.
 
+### Plugin signing
 
+The key pair together with the password required for the plugin signing should be present in the environment.
```

Comment:
Hmmm again... the keys should be present but in the CI right? will there ever realistically be a need for the plugins to be signed on developers' machines?